src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -81,6 +107,48 @@
  *       value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void equals() { // violation, should be annotated with &#64;Override
+ *
+ *     }
+ * }
+ *
+ * interface Test {
+ *
+ *      &#47;** {&#64;inheritDoc} *&#47;
+ *      public abstract void test(); // violation, should be annotated with &#64;Override
+ * }
+ *
+ * class Test extends SuperClass {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test() { // OK, is ignored because class extends other class
+ *
+ *     }
+ * }
+ *
+ * class Test implements SuperInterface {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test() { // OK, is ignored because class implements interface
+ *
+ *     }
+ * }
+ *
+ * class Test {
```

Comment:
Please change class/interface names in this example, they all are "Test".
Please make them different

Reply:
done